store/sqlstore/store.go

```diff
@@ -1371,3 +1371,36 @@ func VersionString(v int) string {
 	major := v / 10000
 	return strconv.Itoa(major) + "." + strconv.Itoa(minor)
 }
+
+func (ss *SqlStore) SetReplicationLagForTesting(seconds int) error {
```

Comment:
It looks like this is only to be used for testing. Wondering if we can move it to a `_test.go` file instead? Otherwise it will be part of the built binary too.